Resources/config/routing.yml

```diff
@@ -1,10 +1,10 @@
 lexik_translation_list:
-    pattern:      /translations
+    pattern:      /list
     defaults:     { _controller: "LexikTranslationBundle:Rest:list" }
     requirements: { _method: GET }
 
 lexik_translation_update:
-    pattern:      /translations/{id}
+    pattern:      /update/{id}
```

Comment:
These two routes aim to be restful, so the url is about what you deal with (translations) and the http method is about what you want to do.
Maybe I should put them in a separated file to make the difference between API and other routes.